### PR TITLE
Fixed files

### PR DESCRIPTION
**Pull request by MrSkorohod**
Reorganizing files. Adding a package.json and gulpfile. Adding classes according to the BEM methodology.

### CODE REVIEW

**Comment by ElizavetaSelivanova** [comment]
Incorrect branch naming.

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 node_modules/
 .idea/
+dist/
```
Comment (ElizavetaSelivanova) [inline .gitignore]: Add an empty line to the end of file

File: stylesheets/base/_typography.scss [hunk 2]
```diff
@@ -6,13 +6,15 @@ body {
   color: $text-color;
   font: 400 14px / 28px $text-font;
 }
-.header-navigation {
+.headerNavigation {
```
Comment (ElizavetaSelivanova) [inline stylesheets/base/_typography.scss]: Classname - try to not use camel-case for classes

File: stylesheets/base/_typography.scss [hunk 3]
```diff
@@ -6,13 +6,15 @@ body {
...
-.logotype h1{
+
+.logotype-header{
   color: $logo-color;
   font: 700 30px / 34px $text-font;
 }
+
 .yellow-button{
   color: $light-grey;
   font: 600 16px / 24px $text-font;
```
Comment (ElizavetaSelivanova) [inline stylesheets/base/_typography.scss]: You can try to add mixins for fonts

File: stylesheets/base/_typography.scss [hunk 4]
```diff
@@ -21,67 +23,62 @@ body {
...
-.name-section {
+
+.designTools-name {
   color: $brand-color;
   font: 700 36px / 54px $text-font;
 }
-.design-tools-cards{
-  h2 {
+
+.design-cards{
```
Comment (ElizavetaSelivanova) [inline stylesheets/base/_typography.scss]: Add space before '{'

File: stylesheets/layout/_footer.scss [hunk 5]
```diff
@@ -23,7 +23,7 @@ h3 {
     margin-bottom: 25px;
     padding-left: 15px;
   }
-  button{
+  .yellow-button{
```
Comment (ElizavetaSelivanova) [inline stylesheets/layout/_footer.scss]: Are you sure this is the correct class name? What happens if the colors will be changed =)

File: stylesheets/layout/_section-design-tools.scss [hunk 6]
```diff
@@ -3,33 +3,33 @@
...
 
-.design-background {
-  background: url("img/Bg.png") no-repeat;
+.designTools-background {
+  background: url("../assets/img/Bg.png") no-repeat;
   background-size: 100% 100%;
   padding-bottom: 1px;
   margin-bottom: 120px;
-  .design-tools-cards{
+  .design-cards{
```
Comment (ElizavetaSelivanova) [inline stylesheets/layout/_section-design-tools.scss]: empty space before new class

File: stylesheets/layout/_section-news.scss [hunk 7]
```diff
@@ -1,20 +1,20 @@
-.section-news {
+.sectionNews {
   padding-bottom: 120px;
   border-bottom: 1px solid $light-grey  ;
```
Comment (ElizavetaSelivanova) [inline stylesheets/layout/_section-news.scss]: Remove spaces before `;`

File: stylesheets/main.scss [hunk 8]
```diff
@@ -23,7 +23,6 @@
   'layout/main-section',
 'layout/section-design-tools',
```
Comment (ElizavetaSelivanova) [inline stylesheets/main.scss]: Fix alignment

**Review by ElizavetaSelivanova — CHANGES_REQUESTED** [review]

**Comment by angelinabylitskaya** [comment]
Closed due to wrong branching strategy.

**Pull request closed without merging**